code/obj/item/organs/head.dm

```diff
@@ -258,7 +258,61 @@
 		. = ..()
 		src.transplanted = 1
 
+	///Taking items off a head
+	attack_self(mob/user as mob)
+		var/list/headwear = list(head, ears, wear_mask, glasses)
+		if (!length(headwear))
+			return
```

Comment:
Pretty sure a list of `null` values still has a length.

Reply:
Indeed, and it turns out I didn't realise the ```if (!selection)``` already did what I was trying to do.